Automattic/woocommerce-payments
# Add SEPA Direct Debit payment method.

Fixes #1468 

Adds the SEPA direct debit payment method. Is hidden behind a feature flag controlled by the `_wcpay_feature_sepa` option.
This MVP implementation and there will be many follow up PRs. This is being merged now to keep the PR smaller.

### Testing instructions
* set store currency to euro
* Enable feature flag wp option update _wcpay_feature_sepa 1
* Enable sepa in checkbox at bottom of /wp-admin/admin.php?page=wc-settings&tab=checkout&section=woocommerce_payments
* Add item to cart and go to checkout
* choose SEPA direct debit payment option
* enter test IBAN DE89370400440532013000
* Place order
* Order should be placed and show the PAYMENT METHOD: SEPA Direct Debit
* Order status should be set to on-hold

### Know Issues to be address seperately
[Transaction details page for SEPA does not load in wcpay transaction tab
](https://github.com/Automattic/woocommerce-payments/issues/1472 )
[Saved CC will be used if selected instead of SEPA even if it's selected
](https://github.com/Automattic/woocommerce-payments/issues/1471)
[Unable to pay for an existing order using SEPA because it says customer email is not provided.
](https://github.com/Automattic/woocommerce-payments/issues/1539)
[SEPA method does not appear on block checkout
](https://github.com/Automattic/woocommerce-payments/issues/1470)
JS code could be better architectures when dealing with 3+ payment methods.
